t/plugin/splunk-hec-logging.t

```diff
@@ -375,3 +375,50 @@ hello world
 the mock backend is hit
 --- no_error_log
 [error]
+
+
+
+=== TEST 11: set route test batched data
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1', ngx.HTTP_PUT, {
+                uri = "/hello",
+                upstream = {
+                    type = "roundrobin",
+                    nodes = {
+                        ["127.0.0.1:1980"] = 1
+                    }
+                },
+                plugins = {
+                    ["splunk-hec-logging"] = {
+                        endpoint = {
+                            uri = "http://127.0.0.1:18088/services/collector",
```

Comment:
where is the backend server ?

Reply:
It is run in CI https://github.com/apache/apisix/blob/1e31e9da12191789ea30439e82d2d45ed50b9b00/ci/pod/docker-compose.plugin.yml#L195

Reply:
https://github.com/apache/apisix/blob/1e31e9da12191789ea30439e82d2d45ed50b9b00/ci/pod/docker-compose.plugin.yml#L195

Reply:
copy